src/iam.ts

```diff
@@ -57,6 +57,7 @@ import { DIDDocumentFull } from '@ew-did-registry/did-document';
 import { Methods } from '@ew-did-registry/did';
 import { addressOf } from '@ew-did-registry/did-ethr-resolver';
 import { isValidDID } from './utils/did';
+import { arrayify, defaultAbiCoder, id, keccak256, solidityKeccak256 } from "ethers/utils";
```

Comment:
I think it's better to import as utils and then reference sub functions: ethers-io/ethers.js#349

Reply:
Fixed